lib/helpers/answerHelper.js

```diff
@@ -77,7 +75,18 @@ function multipleAnswersFormat(sectionObject) {
     return answerInstructions.substring(0, answerInstructions.length - lineEnd.length); // remove trailing break
 }
 
-function summaryFormatter(answerObject) {
+function simplifyUiSchema(uiSchema) {
```

Comment:
Can we remove this function and just use the UISchema? I'm not there is any benefit to doing this, requires additional testing too